ingestion/src/metadata/ingestion/source/database/bigquery.py

```diff
@@ -94,6 +95,18 @@ def get_columns(bq_schema):
 _types.get_columns = get_columns
 
 
+@staticmethod
+def _build_formatted_table_id(table):
+    """We overide the methid as it returns both schema and table name if dataset_id is None. From our
+    investigation, this method seems to be used only in `_get_table_or_view_names()` of bigquery sqalchemy
+    https://github.com/googleapis/python-bigquery-sqlalchemy/blob/2b1f5c464ad2576e4512a0407bb044da4287c65e/sqlalchemy_bigquery/base.py
+    """
```

Comment:
To confirm, we don't want to return
```python
"{}.{}".format(table.reference.dataset_id, table.table_id)
```
when `dataset_id` is not `None`, right?

Reply:
That is my understanding (based on the behavior of other connector) and this code snipet we had where we fetch just the table name.

cc @pmbrull to confirm the assumption

```python
    def standardize_table_name(self, schema: str, table: str) -> str:
         segments = fqn.split(table)
         if len(segments) != 2:
             raise ValueError(f"expected table to contain schema name already {table}")
         if segments[0] != schema:
             raise ValueError(f"schema {schema} does not match table {table}")
         return segments[1]
```

Reply:
yup BQ returns table names weirdly, so we really only want the name and not the `project.name`